src/rules.js

```diff
@@ -222,7 +222,7 @@ inline.strong.start = edit(inline.strong.start)
 
 inline.strong.middle = edit(inline.strong.middle)
   .replace(/punctuation/g, inline._punctuation)
-  .replace(/blockSkip/g, inline._blockSkip)
+  .replace(/overlapSkip/g, inline._overlapSkip)
```

Comment:
this seems to be a typo from #1686

Reply:
Good catch. Did that make any difference on the tests?

Reply:
no it didn't. I think in order for us to pass all of the commonmark specs for em and strong we will need a recursive way to skip internal strong and em. For example `**a **b** c**` should output `<strong>a <strong>b</strong> c</strong>` but it does not. We would need some way to realize that `b**` is the end of the second strong and not the first.

Reply:
To be honest I don't know why anyone would actually do that. I'm ok with saying that is just garbage in garbage out but it is spec'd so we should probably try to find a way to fix it.